### ISSUE TITLE
Validation Extension and Max Size Issue attaching from component

### ISSUE THREAD

**Issue by chrisvidal** [responsiv/uploader-plugin#60]
here is my code from a component:

```php

        $component = $this->addComponent(
            'fileUploader',
            'companyFileUploader', [
                'deferredBinding' => false,
                'placeholderText' => "Click or drag files to upload",
                'maxSize' => "15", // 15 MB
                'fileTypes' => ".pdf",
            ]
        );
        $component->bindModel('attachments', $this->company);
```

And when I try to upload a PDF file from my form:
- I have `validation.extensions` message displayed below the file (which is not uploaded).
- or i have `The file data may not be greater than 5120 kilobytes.` message, which does not take in account the 15MB   (which is not uploaded)

`Responsiv Uploader version 1.0.4`

**Comment by chrisvidal** [comment]
it looks like we can't use the imageuploader and fileuploader components at the same time in the same page.

**Comment by daftspunk** [comment]
This should be fixed in v1.0.5  (Allow multiple uploaders on the same page)

See 3a0df39c24c3d622c9f19024154d0fd524951940
